webpack.config.js

```diff
@@ -189,6 +189,10 @@ function createConfig(isProdBuild, latestBuild) {
             "node_modules/@polymer/font-roboto-local/fonts/roboto/Roboto-Bold.ttf",
         },
       }),
+      new webpack.ProvidePlugin({
```

Comment:
Not a big fan of this as we can no longer easily analyze the size of files, as there might be hidden things included.

Reply:
What is the better solution?